<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notepad</class>
 <widget class="QWidget" name="notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Speaker Recognition</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Enrollment">
    <attribute name="title">
     <string>Enrollment</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>User Info</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>243</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enrollName">
           <property name="placeholderText">
            <string extracomment="Enter your name"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Age:   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="Userage">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Sex:   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="Usersex">
             <item>
              <property name="text">
               <string>Male</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Female</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Userimage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UploadImage">
         <property name="text">
          <string>Upload Photo</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>601</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Voice Enrollment</string>
     </property>
     <widget class="QLabel" name="Animation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="enroll">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>enroll</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enrollFile">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>100</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose File...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filename">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>377</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="enrollRecord">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopEnrollRecord">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Record</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Recognition">
    <attribute name="title">
     <string>Recognition</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>561</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string>Recognition</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Claim your identity:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Claimname"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="recoRecord">
         <property name="text">
          <string>Record</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopRecoRecord">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGraphicsView" name="Verificationimage">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="Username_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EditInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>EditInfo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Clear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGraphicsView" name="UserImage_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Conversation">
    <attribute name="title">
     <string>Conversation Mode</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>221</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Conversation Info</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>191</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Saveas">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>96</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>141</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Current Speaker</string>
     </property>
     <widget class="QGraphicsView" name="UserImage_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLineEdit" name="Username_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>131</width>
        <height>33</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="conversationRecord">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
    <widget class="QLabel" name="Recordtime">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Video">
    <attribute name="title">
     <string>Video Mode</string>
    </attribute>
    <widget class="QMdiArea" name="mdiArea">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>301</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenVideo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>96</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>This page is under construction</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
